components/instructions/programs/associatedTokenAccount.tsx

```diff
@@ -0,0 +1,50 @@
+import { Connection, PublicKey } from '@solana/web3.js'
+import { AccountMetaData } from '@solana/spl-governance'
+import { SPL_TOKENS } from '@utils/splTokens'
+
+const ATA_PROGRAM_ID = new PublicKey(
+  'ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL'
+)
+
+export const ATA_PROGRAM_INSTRUCTIONS = {
+  [ATA_PROGRAM_ID.toBase58()]: {
+    name: 'Associated Token Account Program - Create Associated Token Account',
+    accounts: [
+      'Authority',
+      'Associated Account',
+      'Token Address',
+      'Token Program',
+    ],
+    getDataUI: (
+      _connection: Connection,
+      _data: Uint8Array,
+      accounts: AccountMetaData[]
+    ) => {
+      const ata = accounts[1].pubkey.toString()
+      const tokenMint = accounts[3].pubkey.toString()
+
+      const tokenName = Object.keys(SPL_TOKENS).reduce((tmp, x) => {
+        const { mint, name } = SPL_TOKENS[x]
+
+        if (mint.toString() === tokenMint) {
+          return name
+        }
+
+        return tmp
+      }, 'unknown')
+
+      return (
+        <div style={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
use available style tooling of possible rather than style attribute

Reply:
ok will have to take a look at available styling classes. Keep in mind that the proper way to display theses would be to create a new component (kind of a table). This is best effort

Reply:
`className="flex flex-column"` or something like this probably is already available and is good enough. We don't need to fix the whole codebase, just not make it worse :)